osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -65,56 +65,62 @@ protected override void PopOut()
 
         public void UpdateBeatmap(WorkingBeatmap beatmap)
         {
-            var lastContainer = beatmapInfoContainer;
-            float newDepth = lastContainer?.Depth + 1 ?? 0;
+            LoadComponentAsync(new BufferedWedgeInfo(beatmap)
+            {
+                Shear = -Shear,
+                Depth = info?.Depth + 1 ?? 0,
+            }, newInfo =>
+            {
+                // ensure we ourselves are visible if not already.
+                if (!IsPresent)
+                    this.FadeIn(250);

```

Comment:
Wait what, when does BufferedWedgeInfo even get an alpha of 0? It doesn't seem to ever be?

Reply:
`BeatmapInfoWedge`, not `BufferedWedgeInfo`. `this.FadeIn`.

Reply:
Oops I misread.